main_contrastive.py

```diff
@@ -113,7 +124,7 @@ def parse_args():
     parser.add_argument("--dali_device", type=str, default="gpu")
     parser.add_argument("--last_batch_fill", action="store_true")
 
-    # extra simclr settings
+    # extra simclr/nnclr settings
```

Comment:
[for the future] we can divide settings based on families of methods (contrastive / momentum / clustering methods)

Reply:
I think it would be better to have multiple functions (one per algorithm + dataset related stuff + training and so on) that adds arguments. I think it would be easier to maintain everything